mobile/src/main/java/org/openhab/habdroid/ui/WriteTagActivity.kt

```diff
@@ -259,8 +271,14 @@ class WriteTagActivity : AbstractBaseActivity(), CoroutineScope {
 
             val nfcActivate = view?.findViewById<TextView>(R.id.nfc_activate)
             nfcActivate?.isVisible = true
+
+            val action = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
+                Settings.Panel.ACTION_NFC
```

Comment:
I haven't tested the setting panel as I don't have a device with NFC and Android Q.